c/zss.c

```diff
@@ -727,6 +727,43 @@ static int checkLoggingVerbosity(const char *serverConfigFile, char *pluginIdent
   return ZOWE_LOG_INFO;
 }
 
+static void checkAndSetServiceLoglevel(const char *serverConfigFile, const char *pluginIdentifier, 
+                                       uint64 loggingIdentifier, ShortLivedHeap *slh) {
+  char errorBuffer[1024] = {0};
+  int logLevel = ZOWE_LOG_NA;
+  int identifierLength = strlen(pluginIdentifier);
+  char identifierBuffer[identifierLength+1];
+  memset(&identifierBuffer, 0, sizeof(identifierBuffer));
+  strncpy(identifierBuffer, pluginIdentifier, identifierLength);
+  for (int i = 0; i < identifierLength; i++) {
+    if (identifierBuffer[i] == '/') {
+        identifierBuffer[i] = '.';
+    }
+  } 
+
+  Json *json = jsonParseFile(slh, serverConfigFile, errorBuffer, sizeof(errorBuffer));
```

Comment:
Do you really need to parse the server config file so many times? It would be better to pass `mvdSettings` and `envSettings` into `readWebPluginDefinitions`  as new arguments.

Reply:
Fixed it. 
Environment variable logLevels handled by referring https://github.com/zowe/zlux/wiki/Configuration-for-ZLUX-App-Server-&-ZSS#:~:text=____%20maps%20to%20literal%20.
If logLevels key name is org.zowe.zlux.sample.angular.zssStorage then assumed mapped env variable is ZWED_logLevels_org___zowe___zlux___sample___angular___zssStorage.